src/codesyncd/utils.ts

```diff
@@ -197,11 +198,19 @@ export class statusBarMsgs {
 		if (!fs.existsSync(this.settings.CONFIG_PATH)) return STATUS_BAR_MSGS.NO_CONFIG;
 		const repoPath = pathUtils.getRootPath();
 		const daemonError = CodeSyncState.get(CODESYNC_STATES.DAEMON_ERROR);
-		const userState = new UserState().get();
+		const userState = new UserState();
+		const user = new UserState().get();
+		// Login is in process
+		if (user.isWaitingForLogin) {
+			const authInProcess = CodeSyncState.canSkipRun(CODESYNC_STATES.USER.AUTHENTICATION_INITIATED_AT, AUTHENTICATION_TIMEOUT);
+			if (authInProcess) return STATUS_BAR_MSGS.WAITING_FOR_LOGIN;
+			userState.setInvalidAccount();
```

Comment:
I didn't get this, why are we setting invalid account after auth process completes? What if Auth was successful 
![image](https://github.com/codesyncapp/codesync-vs/assets/15974895/3e2c5bf7-027a-46eb-a1d0-4e8323a88c02)

Reply:
So the msg "Waiting for login confirmation..." will appear only for 5 minutes. After that, we'll consider that user didn't proceed with the Login. 
This is just to hide "Waiting for login confirmation..." after some time, otherwise, it will stay forever if user does not enter credentials to complete Auth.